src/App.js

```diff
@@ -1,8 +1,37 @@
 import './App.css';
+import Tasks from './todo.json'
+import { useState } from "react"
 
 function App() {
-  return (
-    <></>
+
+  const [indexCheck, setIndexCheck] = useState([])
+
+  function change(event, index) {
```

Comment:
@PuniaMohit what do you mean by function name change? also why are you not using es6 functional conventions